src/app/utils/utils.js

```diff
@@ -371,6 +371,73 @@ const getUpdatedFilterValues = (props) => {
   return updatedFilterValues;
 };
 
+function displayContext({ searchKeywords, selectedFilters, field, count }) {
```

Comment:
Since this is removed from its place of intended use, it'll be helpful to put a comment block here describing what it is doing and in what situations someone should use it.